app/src/main/res/values/strings.xml

```diff
@@ -34,7 +34,7 @@
     <string name="keyboard_height">Keyboard height</string>
     <string name="show_key_borders">Show key borders</string>
     <string name="show_numbers_row">Show numbers on a separate row</string>
-    <string name="start_sentences_capitalised">Start sentences with a capital letter</string>
+    <string name="enable_sentences_capitalization">Capisalise the first word of each sentence</string>
```

Comment:
I dont really want to change the string, added it earlier to give more time to translators. And even if you change a string, you should obviously change it in all languages... Revert this line please.

Reply:
Hmm, that's strange because I had no any strings like that, that's why it was added🫣

Reply:
> I dont really want to change the string, added it earlier to give more time to translators. And even if you change a string, you should obviously change it in all languages... Revert this line please.

Seems like it was pulled after I have created a pull request. I will be more careful in future and will update the project every time before starting.